Admin ui features - Mariann- ready to review.md (#72)
SenseNet/docs.sensenet.com: content/guides/setup_settings.md

---
title: "Setup"
metaTitle: "sensenet admin-ui - Setup"
metaDescription: "sensenet admin-ui  -Setup"
---

> 
> # Setup

Reviewer: ```suggestion
```

> With the _Setup_ menu in the Menu bar on the left of the admin-ui che can customize your setting.

Reviewer: ```suggestion
By selecting the _Setup_ option from the left side menu you can customize different settings rigth on the admin-ui.
```

> Opening it you can change for example Portal settings, Indexing, OfficeOnline Settings, ect.

Reviewer: ```suggestion
Here you can modify for example *Portal settings*, *Indexing*, *OfficeOnline Settings* and so on.
```

> ![setup-dashboard](/content/guides/img/setup-dashboard.png)

Reviewer: ```suggestion
![setup-dashboard](./img/setup-dashboard.png)
```

> In sensenet you can make custom setting globally, from the Menu bar on the left, or locally, for example on a file or a folder.

Reviewer: ```suggestion
In sensenet we differentiate local and global settings.
Global settings can be set in the setup menu, local ones in the other hand on a specific file or folder fox example.
```

> To set for example watermark, it is possible to make it globally for each and every preview image from Menu bar/Setup/DocumetPreview.Settings.

Reviewer: ```suggestion
Watermark can be set globally in *DocumentPreview.Settings.* resulting that each and every preview image will have the same watermark.
```

In addition to that, you can define different watermarks for different workspaces (which will then overwrite global settings), or even set watermark to a specific doclib. 
Watermark is just one example of how versatile settings in sensenet can be, mainly due to the hierarchical tree structure, which forms the base of our content repository.

> ![edit_file](/content/guides/img/edit_file.png)

Reviewer: ```suggestion
![edit_file](./img/edit_file.png)
```
